src/Output/Output_DumpData_Total.cpp

```diff
@@ -712,6 +712,7 @@ void Output_DumpData_Total( const char *FileName )
 
 #     ifndef GRAVITY
       const double AveDensity_Init     = NULL_REAL;
+      const double GREP_Center[3]      = { NULL_REAL };
```

Comment:
This will initialize only the first element to `NULL_REAL`. Have you checked the other two values are `NULL_REAL`?

Reply:
You are right. I corrected this.